compiler/bindings/python/test/tools/ir_tool_test.py

```diff
@@ -0,0 +1,114 @@
+# Copyright 2023 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+from iree.compiler.tools.ir_tool import __main__
```

Comment:
This test failed on the Windows CI:
https://github.com/openxla/iree/actions/runs/5829685985/job/15809623394#step:9:2055
```
 962/1162 Test #1331: iree/compiler/bindings/python/test/tools/ir_tool_test ......................................................***Failed    1.12 sec
Traceback (most recent call last):
  File "C:\a\iree\iree\compiler\bindings\python\test\tools\ir_tool_test.py", line 7, in <module>
    from iree.compiler.tools.ir_tool import __main__
  File "C:\a\iree\iree\build-windows\compiler\bindings\python\iree\compiler\tools\ir_tool\__main__.py", line 11, in <module>
    from ...api import Invocation, Session, Source, Output
  File "C:\a\iree\iree\build-windows\compiler\bindings\python\iree\compiler\api\__init__.py", line 26, in <module>
    from .ctypes_dl import (
  File "C:\a\iree\iree\build-windows\compiler\bindings\python\iree\compiler\api\ctypes_dl.py", line 454, in <module>
    _global_init = _GlobalInit()
  File "C:\a\iree\iree\build-windows\compiler\bindings\python\iree\compiler\api\ctypes_dl.py", line 443, in __init__
    _init_dylib()
  File "C:\a\iree\iree\build-windows\compiler\bindings\python\iree\compiler\api\ctypes_dl.py", line 40, in _init_dylib
    dylib_path = _probe_iree_compiler_dylib()
  File "C:\a\iree\iree\build-windows\compiler\bindings\python\iree\compiler\api\ctypes_dl.py", line 438, in _probe_iree_compiler_dylib
    raise ValueError(f"Could not find {dylib_basename} in {paths}")
ValueError: Could not find IREECompiler.dll in [WindowsPath('C:/a/iree/iree/build-windows/lib')]
Exception ignored in: <function _GlobalInit.__del__ at 0x000001A5E8B33520>
Traceback (most recent call last):
  File "C:\a\iree\iree\build-windows\compiler\bindings\python\iree\compiler\api\ctypes_dl.py", line 450, in __del__
    self.local_dylib.ireeCompilerGlobalShutdown()
AttributeError: '_GlobalInit' object has no attribute 'local_dylib'
```

Reply:
Oh, I think I had a patch for that on a branch somewhere but never landed. Will fix.

Reply:
Direct pushed a fix: https://github.com/openxla/iree/commit/8ab9e12b0bf7721a648710efbdb24db41f962c10

Reply:
Thanks!